tern/command_lib/base.yml

```diff
@@ -96,6 +97,13 @@ dpkg:
           - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
           - "for p in $pkgs; do cat /usr/share/doc/$p/copyright; echo LICF; done"
     delimiter: 'LICF'
+  files:
+    invoke:
+      1:
+        container:
+        - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
+        - 'for p in $pkgs; do files=`dpkg-query -L $p`; for file in $files; do if [[ -f $file ]]; then echo $file; fi; done; echo LICF; done'
```

Comment:
Working through program flow here. We print to stdout a list of files for each package, and we delimit using `LICF`. I am having a hard time finding the change that parses through this data. Is it `get_package_files`?